numpy/core/src/npysort/quicksort.c.src

```diff
@@ -107,6 +111,7 @@ quicksort_@suff@(void *start, npy_intp num, void *NOT_USED)
                 *sptr++ = pi - 1;
                 pl = pi + 1;
             }
+            *psdepth++ = cdepth + 1;
```

Comment:
ok maybe lets backup, what do you mean with this statement in the first comment:

    *psdepth++ = ++cdepth;

this is the semantically the same as the code in this diff:

    *psdepth++ = cdepth + 1;

maybe you are looking at another broken part of the diff?